pkg/drivers/mysql/mysql.go

```diff
@@ -44,6 +46,9 @@ var (
 		`CREATE INDEX kine_prev_revision_index ON kine (prev_revision)`,
 		`CREATE UNIQUE INDEX kine_name_prev_revision_uindex ON kine (name, prev_revision)`,
 	}
+	schemaMigrations = []string{
+		`ALTER TABLE kine MODIFY COLUMN id BIGINT UNSIGNED NOT NULL UNIQUE, MODIFY COLUMN create_revision BIGINT UNSIGNED, MODIFY COLUMN prev_revision BITINT UNSIGNED`,
```

Comment:
Is 'AUTO_INCREMENT' assumed on 'NOT NULL' with 'UNIQUE'?

Reply:
My understanding is that when modifying a primary key column, you should explicitly include anything that's implied by  `PRIMARY KEY (id)` being present in the table create statement.

Reply:
This statement has already been validated by Hosted Rancher as per https://github.com/k3s-io/k3s/issues/6176#issuecomment-1920386987

Reply:
I'll add a test that starts up an old version of kine and then runs this migration, just to be sure.